src/bytes.rs

```diff
@@ -1488,7 +1485,7 @@ impl Borrow<[u8]> for BytesMut {
 
 impl fmt::Write for BytesMut {
     fn write_str(&mut self, s: &str) -> fmt::Result {
-        BufMut::put(self, s);
+        self.extend_from_slice(s.as_bytes());
```

Comment:
This has different behavior (`extend_from_slice` grows the buffer but `put` panics on full).

That said, this is probably fine? Thoughts?